int32 temperature
bool tempState
float64 pressure
bool preState
float64 humidity
bool humState
---
string result_message
float64 efficiency